CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 20.06
+
+* New API to send DTMF digits to a call
+
+  * `PUT /calls/{call_id}/dtmf`
+  * `PUT /users/me/calls/{call_id}/dtmf`
```

Comment:
Have you thought how to implement DTMF in the other direction, if we ever need it? For example to send DTMF to a trunk call?